### PR TITLE
Optimize the commands

### PR DESCRIPTION
**Pull request by singularo**
Like teh other

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,51 +9,35 @@ ENV LC_ALL     en_AU.UTF-8
...
-
-# Update again now with the extra repos
-RUN apt-get update
-RUN apt-get -y dist-upgrade
-
-# Install new packages we need
-RUN apt-get -y --force-yes install docker-engine
-
-# Configure timezone
-RUN echo "Australia/Adelaide" > /etc/timezone; dpkg-reconfigure tzdata
```
Comment (pingers) [inline Dockerfile:29]: Do we need this bit?

Comment (singularo) [inline Dockerfile:29]: Well, as this is about local only docker de, no we dont really need to be able to call out to the docker on the host. Is that what you mean?

Comment (pingers) [inline Dockerfile:29]: I'm talking about `echo "Australia/Adelaide" > /etc/timezone; dpkg-reconfigure tzdata`. We're only setting php timezone in the MR.

**Comment by pingers** [comment]
Looks good! :shipit:

**Pull request closed without merging**